tests/core/privilege_rules_test.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from unittest.mock import patch, MagicMock
+from superdesk.errors import SuperdeskApiError
+from superdesk.core.auth.privilege_rules import privilege_based_rules, required_privilege_rule
+
+
+@patch("superdesk.core.auth.privilege_rules.is_admin")
```

Comment:
These 3 would be skipped, as we're using `-p no:asyncio` in the pytest call (asyncio plugin was causing an issue, so I disabled it in superdesk tests).
Instead, you could use the test class' we have in superdesk.tests module